### PR TITLE
Feature: structure and packages updated

### PR DESCRIPTION
**Pull request by HackoDev**
* urls reworked
* drf_yasg, oauth2_provider, rest_framework
* dev compose added
* versions bumped
* debug mode restriction added for non-local env

### CODE REVIEW

File: src/services/celery_tasks/dummy/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from .dummy_task import dummy_task  # noqa: F401
```
Comment (GininDenis) [inline src/services/celery_tasks/dummy/__init__.py:1]: do you rly need this nesting? many dummy tasks expected?

**Review by GininDenis — COMMENTED** [review]
@HackoDev,  do we plan to do smth with tasks autodiscover? it makes sense to write custom fixup for celery i guess(based on the django fixup). this manuall process with every module register is super annoying

**Comment by HackoDev** [comment]
> @HackoDev, do we plan to do smth with tasks autodiscover? it makes sense to write custom fixup for celery i guess(based on the django fixup). this manuall process with every module register is super annoying

@GininDenis , it's a good point, thanks. gonna add it.

File: src/services/celery_tasks/dummy/__init__.py (hunk 1 above)
Comment (HackoDev) [reply src/services/celery_tasks/dummy/__init__.py:1]: probably do not need, just checked we usually create single file for each combined scope of tasks like "stripe.py", "external_apis.py", etc.

**Review by GininDenis — APPROVED** [review]

**Pull request merged**